src/Conf.ml

```diff
@@ -319,6 +319,11 @@ let wrap_comments =
   let default = false in
   mk ~default Arg.(value & flag & info ["wrap-comments"] ~doc ~env)
 
+let break_cases =
+  let doc = "Break pattern match cases." in
+  let default = false in
```

Comment:
@jberdine, what do you think about changing the default to true ?

Reply:
I don't know... choosing defaults is hard. Given the config files, I can change our code bases to disable it if we turn it on by default, of course. My default inclination is to leave the defaults unchanged where the choice seems to be largely one of preference, to avoid unnecessarily churning code where the config files do not specify all settings.

Reply:
> to avoid unnecessarily churning code where the config files do not specify all settings.

I don't think I understand what you mean

Reply:
I'm just thinking that it is in general easier for users if upgrading does not needlessly change behavior, rather than add new behavior that can be enabled asynchronously with upgrading. Probably not a big deal, but that way it is more likely that the version and config files do not need to be changed atomically. Depending on how things are deployed (e.g. if binaries use something like rpms and config files are in source control), that can help.

Reply:
ack